documentation/docs/30-advanced/20-hooks.md

```diff
@@ -4,12 +4,13 @@ title: Hooks
 
 'Hooks' are app-wide functions you declare that SvelteKit will call in response to specific events, giving you fine-grained control over the framework's behaviour.
 
-There are two hooks files, both optional:
+There are three hooks files, all optional:
 
 - `src/hooks.server.js` — your app's server hooks
 - `src/hooks.client.js` — your app's client hooks
+- `src/hooks.js` — your app's isomorphic hooks
```

Comment:
perhaps we should avoid using the term isomorphic here as not everyone would know what it means. I think it's fine that we use it below since we define it there
```suggestion
- `src/hooks.js` — your app's hooks that run on both the client and server
```

alternatively we could link to that section like "your app's [isomorphic hooks](#isomorphic-hooks)", but I think it's easier to just avoid the issue